dapr/clients/grpc/_helpers.py

```diff
@@ -5,15 +5,73 @@
 Licensed under the MIT License.
 """
 
+from collections import namedtuple
 from typing import Dict, List, Union, Tuple
+
+from google.protobuf.any_pb2 import Any as GrpcAny
+from google.protobuf.message import Message as GrpcMessage
 from grpc import UnaryUnaryClientInterceptor, ClientCallDetails     # type: ignore
 
-from collections import namedtuple
 
 MetadataDict = Dict[str, List[Union[bytes, str]]]
 MetadataTuple = Tuple[Tuple[str, Union[bytes, str]], ...]
 
 
+def tuple_to_dict(tupledata: MetadataTuple) -> MetadataDict:
+    """Converts tuple to dict.
```

Comment:
MetadataTuple to MetadataDict

Reply:
MetadataTuple and MetadataDict is alias of the Dict and Tuple. I would use python primitive type in comments for now. we can reiterate the comments later.

Reply:
same here.